[2024-09-03T14:12:31.986+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-09-03T14:12:31.999+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: redshift_etl_dag.extrear_datos_API manual__2024-09-03T14:12:30.593648+00:00 [queued]>
[2024-09-03T14:12:32.006+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: redshift_etl_dag.extrear_datos_API manual__2024-09-03T14:12:30.593648+00:00 [queued]>
[2024-09-03T14:12:32.008+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 6
[2024-09-03T14:12:32.018+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): extrear_datos_API> on 2024-09-03 14:12:30.593648+00:00
[2024-09-03T14:12:32.024+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=119) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-09-03T14:12:32.026+0000] {standard_task_runner.py:63} INFO - Started process 121 to run task
[2024-09-03T14:12:32.026+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'redshift_etl_dag', 'extrear_datos_API', 'manual__2024-09-03T14:12:30.593648+00:00', '--job-id', '5', '--raw', '--subdir', 'DAGS_FOLDER/dag_etl.py', '--cfg-path', '/tmp/tmpidqkejht']
[2024-09-03T14:12:32.028+0000] {standard_task_runner.py:91} INFO - Job 5: Subtask extrear_datos_API
[2024-09-03T14:12:32.070+0000] {task_command.py:426} INFO - Running <TaskInstance: redshift_etl_dag.extrear_datos_API manual__2024-09-03T14:12:30.593648+00:00 [running]> on host cd927b53650d
[2024-09-03T14:12:32.140+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='jose_araya' AIRFLOW_CTX_DAG_ID='redshift_etl_dag' AIRFLOW_CTX_TASK_ID='extrear_datos_API' AIRFLOW_CTX_EXECUTION_DATE='2024-09-03T14:12:30.593648+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-09-03T14:12:30.593648+00:00'
[2024-09-03T14:12:32.142+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-09-03T14:12:38.358+0000] {python.py:237} INFO - Done. Returned value was: None
[2024-09-03T14:12:38.360+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-09-03T14:12:38.368+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=redshift_etl_dag, task_id=extrear_datos_API, run_id=manual__2024-09-03T14:12:30.593648+00:00, execution_date=20240903T141230, start_date=20240903T141232, end_date=20240903T141238
[2024-09-03T14:12:38.405+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-09-03T14:12:38.420+0000] {taskinstance.py:3503} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-09-03T14:12:38.424+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
